libraries/GCS_MAVLink/GCS_MAVLink_Parameters.cpp

```diff
@@ -45,6 +49,7 @@ const struct AP_Param::GroupInfo *GCS::_chan_var_info[MAVLINK_COMM_NUM_BUFFERS];
 #define AP_MAV_DEFAULT_STREAM_RATE_EXTRA3 1
 #define AP_MAV_DEFAULT_STREAM_RATE_PARAMS 10
 #define AP_MAV_DEFAULT_STREAM_RATE_ADSB 0
+#define AP_MAV_DEFAULT_STREAM_RATE_OPTIONS STANDARD_DEFAULT_STREAM_RATE_OPTIONS
```

Comment:
We don't really need to modify this based on vehicles; just have one default somewhere.

The code blocks you're modifying here we'd actually like to go away!